README.md

```diff
@@ -2,24 +2,16 @@
 
 Helper for running nodejs applications on Platform.sh
 
-##Purpose
+## Purpose
 
-Reads Platform.sh configuration from environment and returns a single object
+Reads [Platform.sh configuration](https://docs.platform.sh/development/variables.html#) from environment and returns a single object
```

Comment:
1. You can remove the trailing hash at the end of the URL.
2. Can you add a dot?